api/src/main/scala/hmda/api/HmdaApi.scala

```diff
@@ -36,9 +41,25 @@ object HmdaApi
 
   //Start up API Actors
 
-  createSingleLarValidator(system)
-  createInstitutions(system)
-  createLocalHmdaEventProcessor(system)
+  val supervisor = HmdaSupervisor.createSupervisor(system)
+
+  (supervisor ? FindActorByName(SingleLarValidation.name))
+    .mapTo[ActorRef]
+    .map { actor =>
+      log.info(s"Started validator at ${actor.path}")
+    }
+
+  (supervisor ? FindActorByName(LocalHmdaEventProcessor.name))
+    .mapTo[ActorRef]
+    .map { actor =>
+      log.info(s"Started event processor at ${actor.path}")
+    }
+
+  (supervisor ? FindActorByName(InstitutionPersistence.name))
```

Comment:
I am starting it here on purpose, since `/institutions` is the entry point. It doesn't hurt to have it running, since this actor is unique for the whole application, as opposed to institution specific ones, where there will be thousands of copies being instantiated.